### ISSUE TITLE
Build Issue with Scons - /bin/ld: cannot find -lblas

### ISSUE THREAD

**Issue by rogergermann** [bayerj/arac#27]
Hi Everyone

I get an error building the arac library. The ld command doesn't find the "-lblas". Any ideas what I could do? 

It's running on a CentOS Server.

TIA for any help!

`g++ -o libarac.so -shared src/cpp/arac.os src/cpp/common/buffer.os src/cpp/utilities/utilities.os src/cpp/datasets/sequence.os src/cpp/datasets/supervised.os src/cpp/datasets/unsupervised.os src/cpp/optimization/backprop.os src/cpp/optimization/descent/descender.os src/cpp/optimization/descent/stepdescender.os src/cpp/structure/component.os src/cpp/structure/parametrized.os src/cpp/structure/structure.os src/cpp/structure/connections/blockpermutation.os src/cpp/structure/connections/connection.os src/cpp/structure/connections/connections.os src/cpp/structure/connections/convolve.os src/cpp/structure/connections/full.os src/cpp/structure/connections/identity.os src/cpp/structure/connections/inconvolve.os src/cpp/structure/connections/linear.os src/cpp/structure/connections/outconvolve.os src/cpp/structure/connections/permutation.os src/cpp/structure/modules/bias.os src/cpp/structure/modules/cosine.os src/cpp/structure/modules/doublegate.os src/cpp/structure/modules/errorscaling.os src/cpp/structure/modules/gate.os src/cpp/structure/modules/linear.os src/cpp/structure/modules/lstm.os src/cpp/structure/modules/mdlstm.os src/cpp/structure/modules/module.os src/cpp/structure/modules/modules.os src/cpp/structure/modules/multiplication.os src/cpp/structure/modules/partialsoftmax.os src/cpp/structure/modules/sigmoid.os src/cpp/structure/modules/softmax.os src/cpp/structure/modules/switch.os src/cpp/structure/modules/tanh.os src/cpp/structure/networks/basenetwork.os src/cpp/structure/networks/network.os src/cpp/structure/networks/networks.os src/cpp/structure/networks/mdrnns/basemdrnn.os src/cpp/structure/networks/mdrnns/mdrnn.os -L/usr/lib -L. -L/usr/local/lib -L/sw/lib -lm -lblas
/bin/ld: cannot find -lblas
collect2: error: ld returned 1 exit status
scons: *** [libarac.so] Error 1
`
